include/panning.h

```diff
@@ -40,7 +42,7 @@ inline StereoVolumeVector panningToVolumeVector( panning_t _p,
 {
 	StereoVolumeVector v = { { _scale, _scale } };
 	const float pf = _p / 100.0f;
-	v.vol[_p >= PanningCenter ? 0 : 1] *= 1.0f - qAbs<float>( pf );
+	v.vol[_p >= PanningCenter ? 0 : 1] *= 1.0f - std::abs(pf);
```

Comment:
Sneaking this in I see. 😁

Reply:
Essentially, this commit was necessary so that the code could compile. I'm guessing ``qAbs`` was brought in by the ``QVector`` header.

I'm probably going to do a thorough removal of ``qAbs``, ``qMin``, ``qMax``, etc from the core later :smile:.

Reply:
Ah! Yeah it brought them in then.